src/microbe_stage/editor/CellEditorComponent.cs

```diff
@@ -146,13 +137,40 @@ public partial class CellEditorComponent :
     private VBoxContainer partsSelectionContainer = null!;
     private CollapsibleList membraneTypeSelection = null!;
 
-    private Label sizeLabel = null!;
-    private Label speedLabel = null!;
-    private Label rotationSpeedLabel = null!;
-    private Label hpLabel = null!;
-    private Label storageLabel = null!;
-    private Label generationLabel = null!;
-    private Label totalPopulationLabel = null!;
+    [JsonProperty]
+    [AssignOnlyChildItemsOnDeserialize]
+    private CellStatsIndicator sizeLabel = null!;
+
+    [JsonProperty]
+    [AssignOnlyChildItemsOnDeserialize]
+    private CellStatsIndicator speedLabel = null!;
+
+    [JsonProperty]
+    [AssignOnlyChildItemsOnDeserialize]
+    private CellStatsIndicator rotationSpeedLabel = null!;
+
+    [JsonProperty]
+    [AssignOnlyChildItemsOnDeserialize]
+    private CellStatsIndicator hpLabel = null!;
+
+    [JsonProperty]
+    [AssignOnlyChildItemsOnDeserialize]
+    private CellStatsIndicator storageLabel = null!;
+
+    [JsonProperty]
+    [AssignOnlyChildItemsOnDeserialize]
+    private CellStatsIndicator digestionSpeedLabel = null!;
+
+    [JsonProperty]
+    [AssignOnlyChildItemsOnDeserialize]
+    private CellStatsIndicator digestionEfficiencyLabel = null!;
+
+    [JsonProperty]
+    [AssignOnlyChildItemsOnDeserialize]
+    private CellStatsIndicator generationLabel = null!;
```

Comment:
Why is the generation label a cell stat indicator?

Reply:
Just for convenience, I don't have to preserve previous way of handling cell stats just for this one?

Reply:
Wasn't it just a normal label that is set with a single method? So it was more effort to rewrite this to use the new class rather than leave this alone, or am I missing something?

Reply:
Ah wait, it just clicked it in my head, yeah that's right...